### PR TITLE
Hotfix/register redirect

### PR DESCRIPTION
**Pull request by FredrikAugust**
You should be redirected to the dashboard after registering, if not, you'll be stuck on `/register`, which won't show anything.

### CODE REVIEW

**Comment by NicolajNiels1** [comment]
👀

File: src/components/organism/authentication/Register.tsx [hunk 1]
```diff
@@ -18,8 +19,9 @@ const Register: React.FC = props => {
   }: IOnRegister) => {
     try {
       Perform.doRegister(firstName, lastName, email, password, dispatch);
+      props.history.push('/');
```
Comment (NicolajNiels1) [inline src/components/organism/authentication/Register.tsx]: 🔍👀

File: src/components/organism/authentication/Register.tsx [hunk 2]
```diff
@@ -32,4 +34,4 @@ const Register: React.FC = props => {
   );
 };
 
-export default Register;
+export default withRouter(Register);
```
Comment (NicolajNiels1) [inline src/components/organism/authentication/Register.tsx]: That's nifty! 🧐😛

**Comment by FredrikAugust** [comment]
Yep! You need to do it like that for react to be able to handle going back
and forward correctly :)

**Comment by FredrikAugust** [comment]
It might actually redirect even though the call failed though due to async
🙃. Will check it out tomorrow.

**Pull request merged**